Add Fedora repo

wchen342 opened the pull request:
I created a repo for Fedora and rewrote the `.spec` file using the upstream Fedora `chromium` package, which is a little different from the one in `chromium-privacy-browser`.
Currently it is being built on OBS so it can potentially support OpenSUSE Tumbleweed too, but I do not have a test system so I will keep it in Fedora for now.
If this is accepted then I will transfer the fedora repo into `ungoogled-software`.

Eloston commented:
> If this is accepted then I will transfer the fedora repo into ungoogled-software.

Do you mean if Fedora accepts the package or I accept this PR? I took a look at the repo; I think it's fine to move the repo under ungoogled-software first, then update this PR

wchen342 commented:
> Do you mean if Fedora accepts the package or I accept this PR

I meant you accept the PR. I have moved the repo to `ungoogled-software`. I assigned the `android` team to it for now otherwise I will be locked out of it, so maybe you want to create a new team if necessary.

Also fixed the link to Android repo in `README`.

Eloston reviewed (approved):
LGTM, Thanks :+1:

Eloston commented:
I've also updated the permissions for the repo. I've created a new fedora team.
